modules/core/src/main/java/org/apache/synapse/deployers/InboundEndpointDeployer.java

```diff
@@ -22,18 +22,23 @@
 import org.apache.axis2.deployment.DeploymentException;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.apache.synapse.SynapseHandler;
 import org.apache.synapse.transport.customlogsetter.CustomLogSetter;
 import org.apache.synapse.config.xml.MultiXMLConfigurationBuilder;
 import org.apache.synapse.config.xml.inbound.InboundEndpointFactory;
 import org.apache.synapse.config.xml.inbound.InboundEndpointSerializer;
 import org.apache.synapse.inbound.InboundEndpoint;
+import org.apache.synapse.AbstractExtendedSynapseHandler;
 
 import java.io.File;
+import java.util.Iterator;
+import java.util.List;
 import java.util.Properties;
 
 public class InboundEndpointDeployer extends AbstractSynapseArtifactDeployer {
 
     private static Log log = LogFactory.getLog(InboundEndpointDeployer.class);
+    private static String ARTIFACT_TYPE = "inbound-endpoint";
```

Comment:
Shall we use the same const https://github.com/wso2/wso2-synapse/blob/master/modules/core/src/main/java/org/apache/synapse/SynapseConstants.java#L514 for consitency.

Reply:
We can't use the same constant because we want to display artifact type as "inbound-endpoint" in Grafana.